src/Message/Message.php

```diff
@@ -5,7 +5,7 @@
 /**
  * @author Michaël Gallego
  */
-class Message implements MessageInterface
+class Message implements FifoMessageInterface
```

Comment:
Why? I'd like to keep two interfaces. People that are creating their custom message (implementing MessageInterface) should not have to implement getDeduplicationId and getGroupId if they are using a standard queue.

Our base implementation will have all the getters done though.